cypress/integration/courses.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('courses page', () => {
+  before(() => {
+    cy.loginAndVisit();
+  });
+
+  it('navigates to Courses page from Summary', () => {
+    cy.contains('Courses').click();
+    cy.location().should(loc => {
+      expect(loc.pathname).to.contain('courses');
+    });
+  });
+
+  it('contains a courses table', () => {
+    cy.get('table[course-table]');
+  });
+
+  it('removes ECS courses from 2018-19', () => {
```

Comment:
This test series will begin breaking in a few months I think.
Currently, on login IPA will start you in the academic year 2018-19, but thats only because that is the 'next' academic year. Once fall rolls around I think it will start you in 2019-20, and then all these tests will break.

I suppose our tests should either try to avoid having an opinion about the year, or reliably navigate to the correct year, regardless of where IPA starts you off.

Perhaps the test to ensure the global nav works (load summary, click courses) could be one test. And then another test for courses behavior would do something like cy.loginAndVisit(courses/2018/10)